src/components/UserRegistrationPage.js

```diff
@@ -1,4 +1,65 @@
+import { useEffect, useState } from "react";
+import { useNavigate } from "react-router-dom";
+import { Link } from "react-router-dom";
+
+
 export default function UserRegistrationPage(){
 
-    return(<></>)
+
+    // states for values in the input field
+    const [username, setUser] = useState("");
+    const [password, setPassword] = useState("");
+    const navigate = useNavigate()
+
+    //update state values
+    function updateUser(event){
+        setUser(event.target.value)
+    }
+
+    function updatePassword(event){
+        setPassword(event.target.value)
+    }
+
+    async function createUser(){
+        const user = {id:0,username:username,password:password}
+        console.log(user)
+        const response = await fetch("http://localhost:8080/users",{
+            body:JSON.stringify(user),
+            method:"POST",
+            headers:{
+                "Content-Type":"application/json"
+            }     
+        });
+
+        if(response.status === 200){
+            const body = await response.json();
+            alert(`New user registered with id of ${body.id}`)
+            // Added session storage for registered user and they are then taken to the potlukks page
+            sessionStorage.setItem("user", JSON.stringify(userInfo)); // store user in session storage
```

Comment:
But passed userInfo

I think one or the other should be changed

Reply:
I updated the variable name to be userInfo consistently in the if statement